API/Repositories/Implementations/RepositoryBase.cs

```diff
@@ -62,4 +69,14 @@ public virtual async Task<int> BulkInsertAsync(IEnumerable<T> entities)
     }
 
     public virtual async Task<IEnumerable<T>> GetAllAsync() => await _context.Set<T>().ToListAsync();
+
+    public virtual async Task<int> BulkUpdateAsync(IEnumerable<T> entities)
```

Comment:
See the implementation of `RepositoryBase.UpdateAsync(IEnumerable<T> entities)` in #228. Think this should overload `UpdateAsync` regardless